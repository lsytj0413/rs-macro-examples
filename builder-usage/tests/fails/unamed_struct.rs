use builder_macro::Builder;

#[derive(Builder)]
struct ExampleStruct ();

fn main() {
    println!("Hello, world!");
}