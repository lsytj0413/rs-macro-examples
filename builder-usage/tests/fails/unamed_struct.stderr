error: proc-macro derive panicked
 --> tests/fails/unamed_struct.rs:3:10
  |
3 | #[derive(Builder)]
  |          ^^^^^^^
  |
  = help: message: not implemented: only implemented for structs
